eng/common/scripts/Update-DocsMsConfigFiles.ps1

```diff
@@ -0,0 +1,53 @@
+param(
+  [Parameter(Mandatory = $true)]
+  [string]$DocsRepo,
+  [Parameter(Mandatory = $false)]
+  [string]$PackageSourceOverride,
+  [Parameter(Mandatory = $false)]
+  [switch]$DailyDocs
+  
+)
+Set-StrictMode -Version 3
+. (Join-Path $PSScriptRoot common.ps1)
+. (Join-Path $PSScriptRoot Helpers/Metadata-Helpers.ps1)
+
+function GetOnboardingPackages($moniker) {
+    # Get all metadata jsons
+    $metadataFiles = Get-ChildItem "$DocsRepo/metadata/$moniker/*" -File -Include *.json
+    $csvPackages = Get-CSVMetadata $MetadataUri
+    $metadataObjects = $metadataFiles.foreach({Get-Content $_.FullName -Raw | ConvertFrom-Json})
+
+    # Filter logic
+    if ($moniker -eq 'legacy') {
+        $csvPackages = $csvPackages.Where(
+            {$_.Support -eq 'deprecated' -and `
+            ($_.Hide -ne 'true' -or $_.MSDocService -ne '')}
+        )
+    }
+    elseif($moniker -eq 'latest') {
+        $csvPackages = $csvPackages.Where({
+            $_.Support -ne 'deprecated' -and `
+            ($_.Hide -ne 'true' -and $_.New -eq 'true' -or $_.MSDocService -ne '') -and `
+            $_.VersionGA -ne ''
+        })
+    }
+    else {
```

Comment:
This should probably be `elseif ($moniker -eq 'preview')` and we should throw an error if the `$moniker` is not expected.

Reply:
`$moniker` is what we defined at L45 in this script. There is no such possibilities. If we need version support in future, we probably don't need the `latest` and `preview` anymore.